core/kazoo_documents/src/kzd_resources.erl.src

```diff
@@ -154,6 +155,18 @@ grace_period(Doc, Default) ->
 set_grace_period(Doc, GracePeriod) ->
     kz_json:set_value([<<"grace_period">>], GracePeriod, Doc).
 
+-spec ignore_flags(doc()) -> kz_term:api_boolean().
```

Comment:
These should be included in kzd_resources.erl

Reply:
done